policy/release/test_test.rego

```diff
@@ -435,6 +435,47 @@ test_wrong_attestation_type if {
 	lib.assert_empty(test.deny) with input.attestations as [tr, tr_slsav1]
 }
 
+test_all_image_processed if {
+	# regal ignore:line-length
+	digests_processed := {"image": {"digests": ["sha256:4e388ab32b10dc8dbc7e28144f552830adc74787c1e2c0824032078a79f227fb"]}}
+	images_processed_result := {
```

Comment:
This variable is unused ([checks caught that)](https://github.com/enterprise-contract/ec-policies/pull/952/checks#step:6:29), similarly for the test below